pom.xml

```diff
@@ -316,7 +316,7 @@
                         <configuration>
                             <rules>
                                 <requireMavenVersion>
-                                    <version>3.6.3</version>
+                                    <version>3.8.4</version>
```

Comment:
Looks like this enforce is not used. I'd either use (preferred) it or remove it.

Reply:
@vjuranek Please create a Jira for enforcing

Reply:
https://issues.redhat.com/browse/DBZ-5069